app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
```

Comment:
```suggestion
    people: dict[str, Person] = {}
```
This way

Reply:
if I change and make type annotations I have
ERROR tests/test_main.py - NameError: name 'Person' is not defined
if I import from __future__ annotation ->
FAILED tests/test_main.py::test_person_instance_attribute_wife_and_husband_doesnt_exists - KeyError: 'body'